pom.xml

```diff
@@ -97,7 +97,10 @@ THE SOFTWARE.
       <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
       <project.build.outputEncoding>UTF-8</project.build.outputEncoding>
       <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+      <ant.version>1.9.2</ant.version>
```

Comment:
Why do we need a dependency on Ant now?

Reply:
It was to fix this error, but it's probably not the best way:

```
[INFO] --- maven-enforcer-plugin:3.0.0-M1:enforce (display-info) @ jacoco ---
[INFO] Ignoring requireUpperBoundDeps in org.kohsuke:access-modifier-annotation
[WARNING] Rule 6: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for org.apache.ant:ant:1.8.4 paths to dependency are:
+-org.jenkins-ci.plugins:jacoco:2.3-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-core:2.54
    +-org.apache.ant:ant:1.8.4
and
+-org.jenkins-ci.plugins:jacoco:2.3-SNAPSHOT
  +-org.jenkins-ci.plugins:dashboard-view:2.9.11
    +-org.jenkins-ci.main:maven-plugin:2.13
      +-org.jenkins-ci.main.maven:maven-agent:1.7
        +-org.apache.ant:ant:1.8.4
and
+-org.jenkins-ci.plugins:jacoco:2.3-SNAPSHOT
  +-org.jenkins-ci.plugins:dashboard-view:2.9.11
    +-org.jenkins-ci.main:maven-plugin:2.13
      +-org.jenkins-ci.lib:lib-jenkins-maven-embedder:3.11
        +-org.apache.ant:ant:1.9.2
```